experimental/generator-dotnet-yeoman/generators/app/templates/webapp/Controllers/BotController.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Integration.AspNet.Core;
+
+namespace Microsoft.Bot.Runtime.WebHost.Controllers
```

Comment:
Same here, throughout the web app project, replace instances of 'Microsoft.Bot.Builder.Runtime.WebHost' with <%= botName %>

Reply:
Or, in this case, Microsoft.Bot.Runtime.WebHost since I guess we missed Builder here.